custom/logistics/tasks.py

```diff
@@ -66,9 +73,7 @@ def stock_data_task(domain, endpoint, apis, test_facilities=None):
         # todo: see if we can avoid modifying the list of facilities in place
         if idx == 0:
             facilities = facilities_copy
-    save_stock_data_checkpoint(checkpoint, 'product_stock', 100, 0, start_date, None, False)
-    checkpoint.start_date = None
-    checkpoint.save()
+    save_stock_data_checkpoint(checkpoint, 'product_stock', 100, 0, None, None)
```

Comment:
@czue Date is sent to API in order to determine what data was migrated and from what date we should start next time. Setting this to None at the end of migration will cause fetching all records from API during next migration which is obviously wrong.